contributing-device-support.md

```diff
@@ -0,0 +1,125 @@
+# Contributing new board support to resin.io
+
+Pre-requisites: yocto BSP layer for your particular board, supplying the BSP (board support package). Should be compatible with yocto daisy/fido/jethro releases.
+
+Every repository used to build a resin.io host Operating System is typically named resin-`<board-family>`. For example, consider the [resin-beaglebone][resin-beaglebone repo] repository which can be used to build a resin.io image for
+the BeagleBone Black, BeagleBone Green and BeagleBone Green WiFi boards.
```

Comment:
I can change that if that is the way to take. What is the preferred repo to make an exmaple out of? @shaunmulligan any comments?